.kokoro/publish-docs.sh

```diff
@@ -13,8 +13,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-#!/bin/bash
```

Comment:
Is this safe? My understanding was that `#!/bin/bash` was necessary to determine the interpreter.

Reply:
As this is generated and run by our CI systems I think it is fine. this really is a hint to the OS so it ought to be alright, but agree it seems best to leave it.